src/components/Launch/LaunchForm/LaunchInterruptibleInput.tsx

```diff
@@ -0,0 +1,99 @@
+import { Typography } from '@material-ui/core';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import * as React from 'react';
+import { Protobuf } from 'flyteidl';
+import { useStyles } from './styles';
+import { LaunchInterruptibleInputRef } from './types';
+import { formStrings } from './constants';
+
+const isValueValid = (value: any) => {
+  return value !== undefined && value !== null;
+};
+
+export interface LaunchInterruptibleInputProps {
+  initialValue?: Protobuf.IBoolValue | null;
+}
+
+export const LaunchInterruptibleInputImpl: React.ForwardRefRenderFunction<
+  LaunchInterruptibleInputRef,
+  LaunchInterruptibleInputProps
+> = (props, ref) => {
+  const [interruptible, setInterruptible] = React.useState(false);
+  const [indeterminate, setIndeterminate] = React.useState(true);
```

Comment:
nit: I think a small comment what `indeterminate` means/represents here could be helpful for future developer's sake.